### ISSUE TITLE
Generating ISBN & ISSN barcodes using the `addontextfont` flag to change fonts (via zCommand Line)

### ISSUE THREAD

**Issue by au-ganymede** [metafloor/bwip-js#341]
## Font change

I am trying to generate an **SVG** file with an **ISBN** barcode and change the default font to my own. 

While `﻿textfont` flag changes the numbers in the barcode, the `﻿addontextfont` flag seems to do nothing. The text above the barcode (ISBN 978-83-8331-461-7) is still rendered using the OCR-A font.

<img src="https://github.com/metafloor/bwip-js/assets/115064696/43a14d57-ef3e-4984-9e88-19536ffea662"  width="300">

## ISSN additional barcode

My second problem is that I cannot generate an ISSN code with an additional number, e.g., `0311-175X 00 17`
I receive the message ﻿`Error: bwipp.issnBadWilndth#5856: ISSN must be 8 or 9 characters including dash, in the format XXXX-XXXX`

However, on the page https://bwip-js.metafloor.com/demo/demo.html entry ﻿`0311-175X 00 17` works:

<img src="https://github.com/metafloor/bwip-js/assets/115064696/e2bf645f-a123-443a-8a9d-b662b581f98b" width="660">

**I would appreciate any advice.**

---

**Used command (ISBN):**
```bash
bwip-js isbn 978-83-8331-461-7 includetext textfont=CONS addontextfont=CONS loadfont=CONS,/Users/ganymede/Library/Fonts/Inconsolata.otf barcode-isbn.svg
```
﻿**bwip-js version:** `4.3.2`

**Comment by metafloor** [comment]
For the ISBN issue, you are confusing the option names.  `addon` is for the supplementary barcode.  You need to use the `isbn` prefix for the options that affect the text above the primary barcode:

```
bwip-js.js isbn "978-83-8331-461-7 07" includetext textfont=CONS isbntextfont=CONS loadfont=CONS,fonts/Inconsolata.otf isbn.svg
```

For the ISSN issue, you will need to provide code that reproduces the error.  If you look at the upstream postscript code, you can see where it breaks text like "0311-175X 00 17" into the three respective pieces:

```
    % Split off the ISSN
    issntxt ( ) search {
        /issntxt exch def
        pop
        /seqvar exch def
    } {
        pop
        /seqvar (00) def
    } ifelse

    % Split off the sequence variant and addon
    seqvar ( ) search {
        /seqvar exch def
        pop
        /addon exch def
    } {
        pop
        /addon () def
    } ifelse
```

The code is using ASCII space as the delimiter between text segments.  I suspect you are using something other than space as the delimiter.

**Comment by au-ganymede** [comment]
Thank you for the explanation. Previously, I typed ﻿`bwip-js --help` but could not find the `﻿isbntextfont` flag (maybe I am blind), so I tried using `﻿addontextfont`.

After your corrections, the text above the barcode is now written in the font I chose. Thank you very much for your help.

Based on the code and explanations you provided about ISSN, I managed to generate a correct barcode. Once again, thank you for the assistance. I appreciate it.
